include/fc/io/cfile.hpp

```diff
@@ -27,14 +27,16 @@ class cfile_datastream;
 class cfile {
 public:
    cfile()
-     : _file(nullptr, &fclose)
+     : _open(false),
+       _file_path(""),
+       _file(nullptr, &fclose)
    {}
 
-   void set_file_path( fc::path file_path ) {
+   void set_file_path( const fc::path &file_path ) {
```

Comment:
The std::move of a const& results in a copy of an lvalue. This is no different from stating "_file_path = file_path". It may also confuse the reader into thinking rvalue move semantics are being implemented.